src/main/java/org/terasology/servermotd/ServerMOTDSystem.java

```diff
@@ -16,30 +16,55 @@
 package org.terasology.servermotd;
 
 import org.terasology.context.Context;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
-import org.terasology.network.NetworkMode;
-import org.terasology.network.NetworkSystem;
+import org.terasology.logic.console.commandSystem.annotations.Command;
+import org.terasology.logic.console.commandSystem.annotations.CommandParam;
+import org.terasology.logic.console.commandSystem.annotations.Sender;
 import org.terasology.registry.CoreRegistry;
-import org.terasology.registry.Share;
+import org.terasology.registry.In;
+
+
+@RegisterSystem(RegisterMode.AUTHORITY)
+public class ServerMOTDSystem extends BaseComponentSystem {
+    @In
+    private EntityManager entityManager;
 
-@Share(ServerMOTD.class)
-@RegisterSystem(RegisterMode.ALWAYS)
-public class ServerMOTDSystem extends BaseComponentSystem implements ServerMOTD {
     private Context context = CoreRegistry.get(Context.class);
 
-    @Override
+    private EntityRef MOTDEntity;
+    private MOTDProvider renderMOTD = new MOTDProvider();
+
     public void initialise() {
-        String motd = "default";
 
-        NetworkSystem networkSystem = context.get(NetworkSystem.class);
+    }
 
-        if (networkSystem.getMode() == NetworkMode.CLIENT) {
-            if (motd != null && motd.length() != 0) {
-                MOTDProvider renderMOTD = new MOTDProvider();
-                renderMOTD.display(motd, context);
-            }
+    @Override
+    public void postBegin() {
+        MOTDEntity = renderMOTD.getMOTDEntity(entityManager);
+    }
+
+    @Command(shortDescription = "Edit server MOTD", helpText = "Edit the current server MOTD if you are admin." +
+            " Use \'a\' - to append to current MOTD" +
+            " 'w' for a new one", runOnServer = true)
+    public String editMOTD(@CommandParam(value = "a/w") String type, @CommandParam(value = "New Message") String message, @Sender EntityRef admin) {
```

Comment:
You'll probably want to move this command into the client system, as otherwise a *headless* server will never be able to have the MOTD edited. Having this command in an authority system means it'll only ever be available if a player is hosting a game directly via the game client. Instead you can use command permissions to ensure only admins can run it (for now: easiest to just require CHEAT level permission to execute)

Also: you might want to accept a single String argument version of the command that assumes the edit mode to be overwrite. You can make a two-arg command variant that accepts a boolean or the existing a/w thing to then select explicitly whether to overwrite or append